.github/workflows/ubuntu.yml

```diff
@@ -107,8 +107,11 @@ jobs:
         id: enable-launchable
         uses: ./.github/actions/launchable/setup
         with:
+          os: ubuntu-20.04
+          test-opts: ${{ matrix.configure }}
           launchable-token: ${{ secrets.LAUNCHABLE_TOKEN }}
           srcdir: src
+          builddir: build
```

Comment:
The order of the options is os, test-opts, launchable-token, builddir, srcdir. We can swap the builddir with srcdir here.